tutorials/rendering/multiple_resolutions.rst

```diff
@@ -265,17 +281,17 @@ what the **Stretch** options above already provide. The default value of ``1.0``
 means that no additional scaling occurs.
 
 For example, if you set **Scale** to ``2.0`` and leave **Stretch Mode** on
-**Disabled**, each unit in your scene will correspond to 2×2 pixels on the
-screen. This is a good way to provide scaling options for non-game applications.
+**Disabled**, each unit in your scene will correspond to 2×2 pixels in the
+window. This is a good way to provide scaling options for non-game applications.
 
-If **Stretch Mode** is set to **canvas_items**, 2D elements will be scaled
-relative to the base window size, then multiplied by the **Scale** setting. This
-can be exposed to players to allow them to adjust the automatically determined
-scale to their liking, for better accessibility.
+If **Stretch Mode** is set to **Canvas Items**, 2D elements will be scaled
```

Comment:
```suggestion
If **Stretch Mode** is set to ``canvas_items``, 2D elements will be scaled
```
This seemed right before, assuming we're talking about how it appears in the project settings. Though `code` style is preferred to **bold** here.